src/styles/blocks.scss

```diff
@@ -0,0 +1,48 @@
+@for $index from 1 to 32 {
+  .day-#{$index} {
+    position: relative;
+    box-sizing: border-box;
+    width: #{$square-size};
+    height: #{$square-size};
+    background-color: #eee;
+    border: 1px solid black;
+    transition: all ease 0.5s;
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+  }
+  
+  .day-#{$index}::before {
+    content: "#{$index}";
+    position: absolute;
+    left: 0;
+    right: 0;
+    display: block;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    text-align: center;
+    line-height: #{$square-size};
+  }
+}
+
+@each $name-day in mon, tue, wed, thu, fri, sat, sun {
+  .calendar__day--start-day--#{$name-day} {
+    margin-left: #{$first-element-margin};
+  }
+}
+
+@for $last-number from 28 to 32 {
+  .calendar__day--month-length--#{$last-number}:nth-child(#{$last-number}) {
```

Comment:
use SCSS nesting:
all styles for calendar elements should be written inside `.calendar` selector:
```
.calendar {
  // own styles
  
  &__element {
    //element styles
  }
}
```